tests/Unit/Evolution/Systems/Cce/Test_LinearSolve.cpp

```diff
@@ -306,6 +307,25 @@ void test_pole_integration_with_linear_operator(
       tmpl::list<BondiValueTag, Tags::PoleOfIntegrand<BondiValueTag>,
                  Tags::LinearFactor<BondiValueTag>,
                  Tags::LinearFactorForConjugate<BondiValueTag>>{});
+  // it is a more realistic linear solve if the linear factors have the right
+  // leading behavior
```

Comment:
I'm not quite sure what you mean by "realistic" here. Do you mean that this more closely resembles the linear solves done in the CCE code? or just that it is mathematically better matching some assumptions? I think if you could clarify this and also where "the right leading behavior" comes from, the comment would be more helpful.